config/manager/controller-config.yaml

```diff
@@ -13,10 +13,10 @@ data:
     progressive:
       defaultAssessmentSchedule:
         - kind: Pipeline
-          schedule: "120,60,10"
+          schedule: "120,360,60,10"
         - kind: MonoVertex
-          schedule: "120,60,10"
+          schedule: "120,360,60,10"
         - kind: InterstepBufferService
-          schedule: "0,0,10"
+          schedule: "0,480,0,10"
```

Comment:
Actually, are we using the 480 or the 60? If not, can we just set those both to 0 for now? (I realize this was already an issue in the existing code)

Reply:
I see that I wrote 480 in the original issue, but i'm not sure why. Maybe I was thinking about in the future if we use it?

Reply:
Yes, Currently we are not using it. For now I'll set it to `0` and once we use it, we can update it